tajo-core/src/main/java/org/apache/tajo/engine/planner/physical/DistinctGroupbySortAggregationExec.java

```diff
@@ -172,6 +168,13 @@ public void close() throws IOException {
 
   @Override
   public void init() throws IOException {
+    super.init();
+
+    if (aggregateExecs != null) {
```

Comment:
@charsyam 
I refer to close() but It seems to the unnecessary checking. Are you working on this operator ?